toc/working-groups/002-PRs-Require-2-Reviewers.md

```diff
@@ -0,0 +1,17 @@
+# PRs Require Two Reviewers
+
+Date of Forum to be Discussed:
+Author: Amelia Downs - @ameowlia
+
+## Problem
+We have more approvers now (a great problem to have)! Let's make sure we keep a transparent and high level
+of code quality.
+
+## Solution
+1. Each PR requires 2 points before it can be merged.
```

Comment:
Looking at what e.g. the Istio community does, we could also allow `lgtm` votes from working group members and require an approval from an approver. That way feedback from multiple people would be required, but only one had to be an approver.

Reply:
If we use github reviews you are not allowed to review your own PR. Let's stick to that since that is how the tool works, plus it means no one can merge their own code with no review.